test/src/test/java/lib/form/ComboBoxTest.java

```diff
@@ -133,4 +139,39 @@ public void testCompoundFieldDependentComboBox() throws Exception {
             Publisher.all().remove(d2);
         }
     }
+
+    public static class XssProperty extends OptionalJobProperty<Job<?,?>> {
+        @TestExtension("testEnsureXSSnotPossible")
+        public static class DescriptorImpl extends OptionalJobProperty.OptionalJobPropertyDescriptor {
+
+            @Override
+            public String getDisplayName() {
+                return "XSS Property";
+            }
+
+            public ComboBoxModel doFillXssItems() {
+                return new ComboBoxModel("<h1>HACK</h1>");
+            }
+        }
+    }
+
+    @Issue("SECURITY-1525")
+    public void testEnsureXSSnotPossible() throws Exception {
```

Comment:
```suggestion
    public void testEnsureXssNotPossible() throws Exception {
```

Excess Snot? No thanks.